libbeat/common/kubernetes/metadata.go

```diff
@@ -54,11 +54,11 @@ func (g *metaGenerator) PodMetadata(pod *Pod) common.MapStr {
 	}
 
 	if len(labelMap) != 0 {
-		meta["labels"] = labelMap
+		meta["labels"] = common.DeDotJSON(labelMap)
```

Comment:
@exekias This is a breaking change. I wonder if you see an issue in this?

Reply:
We need this as a bugfix anyway, isn't it? Didn't follow previous conversations but I guess this is needed to avoid mapping issues?

In any case it's a breaking change and we should make it clear in the release notes

Reply:
As far as I know it didn't break yet but it would pretty soon :-) So yes having it as breaking change caused by a bug is a good way to state it.